Tests/Get-BicepConfig.Tests.ps1

```diff
@@ -0,0 +1,168 @@
+                                "activeDirectoryAuthority": "https://login.chinacloudapi.cn"
+                            },
+                            "AzureUSGovernment": {
+                                "resourceManagerEndpoint": "https://management.usgovcloudapi.net",
+                                "activeDirectoryAuthority": "https://login.microsoftonline.us"
+                            }
+                        },
+                        "credentialPrecedence": [
+                            "AzureCLI",
+                            "AzurePowerShell"
+                        ]
+                    },
+                    "moduleAliases": {
+                        "ts": {},
+                        "br": {
+                            "public": {
+                                "registry": "mcr.microsoft.com",
+                                "modulePath": "bicep"
+                            }
+                        }
+                    },
+                    "analyzers": {
+                        "core": {
+                            "verbose": false,
+                            "enabled": true,
+                            "rules": {
+                                "no-hardcoded-env-urls": {
+                                    "level": "warning",
+                                    "disallowedhosts": [
+                                        "api.loganalytics.io",
+                                        "asazure.windows.net",
+                                        "azuredatalakeanalytics.net",
+                                        "azuredatalakestore.net",
+                                        "batch.core.windows.net",
+                                        "core.windows.net",
+                                        "database.windows.net",
+                                        "datalake.azure.net",
+                                        "gallery.azure.com",
+                                        "graph.windows.net",
+                                        "login.microsoftonline.com",
+                                        "management.azure.com",
+                                        "management.core.windows.net",
+                                        "region.asazure.windows.net",
+                                        "trafficmanager.net",
+                                        "vault.azure.net"
+                                    ],
+                                    "excludedhosts": [
+                                        "schema.management.azure.com"
+                                    ]
+                                },
+                                "no-unused-params": {
+                                    "level": "off"
+                                }
+                            }
+                        }
+                    }
+                }       
+'@
+        }
+
+        It 'Returns default when used without parameters' {
```

Comment:
In the description of the PR (and the stream where you built this) you had an excellent list of all possible scenarios.
Would it be possible to include those scenarios in the it block descriptions here?
something like
'Given path x and no switch statement it should y'
'Given path x and switch statement A it should z'
'Given no path and switch statement A it should F'
This would make it easier to follow that we have all scenarios covered in the future as well.

Reply:
All these scenarios are covered in the assertions below.